ts/features/fims/history/screens/HistoryScreen.tsx

```diff
@@ -0,0 +1,37 @@
+import { VSpacer } from "@pagopa/io-app-design-system";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import * as React from "react";
+import { SafeAreaView, Text, View } from "react-native";
+import LoadingScreenContent from "../../../../components/screens/LoadingScreenContent";
+import { useHeaderSecondLevel } from "../../../../hooks/useHeaderSecondLevel";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { fimsHistoryGet } from "../store/actions";
+import { fimsHistoryPotSelector } from "../store/selectors";
+
+export const FimsHistoryScreen = () => {
+  const dispatch = useIODispatch();
+  const historyPot = useIOSelector(fimsHistoryPotSelector);
+
+  React.useEffect(() => {
+    dispatch(fimsHistoryGet.request({}));
+  }, [dispatch]);
+
+  useHeaderSecondLevel({
+    title: "History"
```

Comment:
We should also add the option to enable the '?' support

Reply:
yes, it will be added once the UI is implemented.